src/test/java/com/google/cloud/tools/maven/configs/DeployConfigTest.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.tools.maven.configs;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Unit tests for {@link DeployConfig}.
+ */
+@RunWith(JUnit4.class)
+public class DeployConfigTest {
+
+  @After
+  public void tearDown() {
+    System.clearProperty("gcp.app.deploy.bucket");
```

Comment:
Unfortunately, Maven relies on these parameters being passed as Java system properties.

I added 
@FixMethodOrder(MethodSorters.NAME_ASCENDING)
to before the class to make sure they run sequentially.